bundler/exe/bundle

```diff
@@ -7,39 +7,26 @@ Signal.trap("INT") do
   exit 1
 end
 
-base_path = File.expand_path("../lib", __dir__)
-
-if File.exist?(base_path)
-  require_relative "../lib/bundler"
-else
-  require "bundler"
-end
+require "bundler"
 
 # Workaround for non-activated bundler spec due to missing https://github.com/rubygems/rubygems/commit/4e306d7bcdee924b8d80ca9db6125aa59ee4e5a3
 gem "bundler", Bundler::VERSION if Gem.rubygems_version < Gem::Version.new("2.6.2")
 
-# Check if an older version of bundler is installed
-$LOAD_PATH.each do |path|
-  next unless path =~ %r{/bundler-0\.(\d+)} && $1.to_i < 9
-  err = String.new
-  err << "Looks like you have a version of bundler that's older than 0.9.\n"
-  err << "Please remove your old versions.\n"
-  err << "An easy way to do this is by running `gem cleanup bundler`."
-  abort(err)
+if RUBY_VERSION < "2.6.0.a" && !ENV["BUNDLE_NO_OLD_RUBIES_WARNING"]
```

Comment:
Can we document this at https://github.com/rubygems/rubygems/blob/master/bundler/lib/bundler/man/bundle-config.1.ronn#list-of-available-keys?

Reply:
I wasn't sure to make this official, the only reason I added it is because otherwise making CI green is going to be really painful, since we constantly check output.

I guess if we want to make it official we can mention in in the warning and that would be enough?

Reply:
sure

Reply:
I changed the name so that the environment variable is not listed with settings, since it's not a setting. Do you think it's necessary to advertise the environment variable, then?

Reply:
Let's leave this as is, if the need to document the environment variable appears later, we can always do that in later iterations.